packages/x-tree-view/src/RichTreeView/RichTreeView.tsx

```diff
@@ -86,6 +95,14 @@ const RichTreeView = React.forwardRef(function RichTreeView<
     ownerState: props as RichTreeViewProps<any, any>,
   });
 
+  if (isLoading) {
```

Comment:
This is really basic :laughing: 
But since it's not the main use case, I think we can release as is, and quickly add a slot for the loading state and another for the error state.